yb-voyager/src/sqlldr/sqlldr.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package sqlldr
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func CreateSqlldrDir(exportDir string) error {
+	if _, err := os.Stat(fmt.Sprintf("%s/sqlldr", exportDir)); os.IsNotExist(err) {
+		err = os.Mkdir(fmt.Sprintf("%s/sqlldr", exportDir), 0755)
+		if err != nil {
+			return fmt.Errorf("create sqlldr directory %q: %w", fmt.Sprintf("%s/sqlldr", exportDir), err)
+		}
+	}
+	return nil
+}
+
+func CreateSqlldrControlFile(exportDir string, tableName string, sqlldrConfig string, fileName string) (sqlldrControlFilePath string, err error) {
+	sqlldrControlFileName := fmt.Sprintf("%s-%s.ctl", tableName, fileName)
+	sqlldrControlFilePath = fmt.Sprintf("%s/sqlldr/%s", exportDir, sqlldrControlFileName)
+	sqlldrControlFile, err := os.Create(sqlldrControlFilePath)
+	if err != nil {
+		return "", fmt.Errorf("create sqlldr control file %q: %w", sqlldrControlFilePath, err)
+	}
+	defer sqlldrControlFile.Close()
+	_, err = sqlldrControlFile.WriteString(sqlldrConfig)
+	if err != nil {
+		return "", fmt.Errorf("write sqlldr control file %q: %w", sqlldrControlFilePath, err)
+	}
+	return sqlldrControlFilePath, nil
```

Comment:
nit: you can use `return` without specifying the vars or if you are doing that remove return variable names from function definition like:
`func CreateSqlldrControlFile(exportDir string, tableName string, sqlldrConfig string, fileName string) (string, error)`

similarly for other functions
Refer: https://www.golangprograms.com/the-return-values-of-a-function-can-be-named-in-golang.html#:~:text=Golang%20allows%20you%20to%20name,value%20that%20the%20function%20returns.